src/KubernetesClient.Models/KubernetesYaml.cs

```diff
@@ -15,14 +15,15 @@ namespace k8s
     /// </summary>
     public static class KubernetesYaml
     {
-        private static readonly DeserializerBuilder CommonDeserializerBuilder =
+        private static DeserializerBuilder CommonDeserializerBuilder =>
```

Comment:
have to `new` every time?

Reply:
Yea, without this the tests fail. This code is only ran 2x, once to new up the StrictDeserializer and once for the normal Deserializer.
![image](https://github.com/kubernetes-client/csharp/assets/9521987/aa29b23d-5915-4dab-b246-9682b0a9ae08)